### PR TITLE
Fix TFM Regression test failures

### PR DESCRIPTION
**Pull request by oyvindronningstad**
The initial attestation tests need tfm_plat_get_initial_attest_key() to be implemented.

### CODE REVIEW

**Comment by ioannisg** [comment]
@andvib could you verify this please?

**Review by hakonfam — APPROVED** [review]

**Review by andvib — APPROVED** [review]
Tested, fixes tests

File: modules/tfm/tfm/boards/common/crypto_keys.c [hunk 1]
```diff
@@ -37,18 +37,54 @@ enum tfm_plat_err_t tfm_plat_get_huk_derived_key(const uint8_t *label,
...
+		return TFM_PLAT_ERR_INVALID_INPUT;
 	}
 
 	int err  = hw_unique_key_derive_key(HUK_KEYSLOT_MKEK, context,
 			context_size, label, label_size, key, key_size);
 
+	if (err) {
+		return TFM_PLAT_ERR_SYSTEM_ERR;
+	}
 	return err;
```
Comment (ioannisg) [inline modules/tfm/tfm/boards/common/crypto_keys.c:49]: Should this return _SUCCESS, then, @oyvindronningstad ?

**Review by ioannisg — COMMENTED** [review]
I have a single comment @oyvindronningstad , thanks for the fix, otherwise.
@andvib thanks for testing

File: modules/tfm/tfm/boards/common/crypto_keys.c (hunk 1 above)
Comment (oyvindronningstad) [reply modules/tfm/tfm/boards/common/crypto_keys.c]: Thanks, done.

**Review by ioannisg — APPROVED** [review]

**Pull request merged**